fetch/src/4_table.r

```diff
@@ -0,0 +1,4 @@
+table_data <- function() {
```

Comment:
I think we could move this step into the `2_process` phase. Then we could read in the data saved here in the 3_visualize `plot_data()` step